definitions/tools/check_strand.cwl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "how_are_we_stranded_here"
```

Comment:
maybe add slightly more info?
`label: "runs how_are_we_stranded_here to determine RNAseq data strandedness"`